serv/algo.py

```diff
@@ -170,9 +180,14 @@ def find_phrase(self, phrase_embed, k_val=5, params=None):
         top_idxs = np.argsort(-np.max(scores, axis=1))[:k_val] #Get max of all chunks in kword & neg for descending
         top_scores = np.sort(-np.max(scores, axis=1))[:k_val]
         top_k_urls = []
+        print("+++++++++++++++++++++++++++++++++++")
+        print(self.rev_data, top_idxs)
         for idx, item in enumerate(top_idxs):
-            print(str(idx) + " " + str(top_scores[idx]) + " " + self.rev_data[item])
-            top_k_urls.append((self.rev_data[item], ''))       # TODO: Update the '' with actual title
+            top_k_urls.append((self.rev_data[item], ''))
+
+        # for idx, item in enumerate(top_idxs):
+        #     print(str(idx) + " " + str(top_scores[idx]) + " " + self.rev_data[item])
+        #     top_k_urls.append((self.rev_data[item], ''))       # TODO: Update the '' with actual title
```

Comment:
Adding here for pkl file added (embed_prod_v02_rc.pkl), remove that. Since file is empty, I can't add comment there.